versionedRouter.js

```diff
@@ -514,24 +517,43 @@ if (startSourceTransformer) {
   });
 }
 
-async function handleDestinationNetwork(version, ctx) {
-  const { destination } = ctx.request.body;
+function handleResponseTransform(version, destination, ctx) {
   const destNetHandler = getDestNetHander(version, destination);
   // flow should never reach the below (if) its a desperate fall-back
-  if (!destNetHandler || !destNetHandler.sendData) {
+  if (!destNetHandler || !destNetHandler.responseTransform) {
     ctx.status = 404;
     ctx.body = `${destination} doesn't support transformer proxy`;
```

Comment:
Can we change `transformer proxy` to `response transformation` ?

Reply:
Yep